core/templates/components/state-editor/state-solution-editor/state-solution-editor.component.html

```diff
@@ -17,7 +17,7 @@ <h3 class="oppia-exp-solution-card-header">Solution</h3>
         </div>
         <div class="state-solution-content-container oppia-mobile-collapsible-card-content" *ngIf="solutionCardIsShown">
           <div class="oppia-add-solution-button-container">
-            <div *ngIf="isEditableOutsideTutorialMode()">
+            <div *ngIf="isEditable()">
```

Comment:
Just to check, does the tutorial still work fully (as it did before)?

Reply:
Yes, I played through the tutorial and it seems to work fine.